### ISSUE TITLE
It's been 2 days since the action is failing (used to work fine)

### ISSUE THREAD

**Issue by claudioandre-br** [actions/first-interaction#286]
It's been 2 days since the `actions/first-interaction` has been failing. Nothing has changed on my end and the missing module looks like something related to deprecation, misuse of something, or a problem in the Ubuntu image itself (BTW: I'm using ubuntu-latest).

```
Run actions/first-interaction@1dbfe1ba5525b8257e1f259b09745bee346d62d8
[...]
internal/modules/cjs/loader.js:1032
2023-11-26T11:24:00.0986995Z   throw err;
2023-11-26T11:24:00.0987957Z   ^
2023-11-26T11:24:00.0988354Z 
2023-11-26T11:24:00.0988907Z Error: Cannot find module 'node:assert'
2023-11-26T11:24:00.0991231Z Require stack:
2023-11-26T11:24:00.0994580Z - /node_modules/undici/lib/handler/RetryHandler.js
2023-11-26T11:24:00.0995171Z - /node_modules/undici/index.js
2023-11-26T11:24:00.0995684Z - /node_modules/@actions/http-client/lib/index.js
2023-11-26T11:24:00.0996348Z - /node_modules/@actions/core/lib/oidc-utils.js
2023-11-26T11:24:00.0996899Z - /node_modules/@actions/core/lib/core.js
2023-11-26T11:24:00.0997346Z - /lib/main.js
2023-11-26T11:24:00.0997969Z     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
2023-11-26T11:24:00.0998746Z     at Function.Module._load (internal/modules/cjs/loader.js:898:27)
2023-11-26T11:24:00.0999399Z     at Module.require (internal/modules/cjs/loader.js:1089:19)
2023-11-26T11:24:00.1000071Z     at require (internal/modules/cjs/helpers.js:73:18)
2023-11-26T11:24:00.1000810Z     at Object.<anonymous> (/node_modules/undici/lib/handler/RetryHandler.js:1:16)
2023-11-26T11:24:00.1001507Z     at Module._compile (internal/modules/cjs/loader.js:1200:30)
2023-11-26T11:24:00.1002206Z     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
2023-11-26T11:24:00.1002871Z     at Module.load (internal/modules/cjs/loader.js:1049:32)
2023-11-26T11:24:00.1003477Z     at Function.Module._load (internal/modules/cjs/loader.js:937:14)
2023-11-26T11:24:00.1004130Z     at Module.require (internal/modules/cjs/loader.js:1089:19) {
2023-11-26T11:24:00.1004696Z   code: 'MODULE_NOT_FOUND',
2023-11-26T11:24:00.1005060Z   requireStack: [
2023-11-26T11:24:00.1005864Z     '/node_modules/undici/lib/handler/RetryHandler.js',
2023-11-26T11:24:00.1006458Z     '/node_modules/undici/index.js',
2023-11-26T11:24:00.1006975Z     '/node_modules/@actions/http-client/lib/index.js',
2023-11-26T11:24:00.1007595Z     '/node_modules/@actions/core/lib/oidc-utils.js',
2023-11-26T11:24:00.1008382Z     '/node_modules/@actions/core/lib/core.js',
2023-11-26T11:24:00.1008832Z     '/lib/main.js'
2023-11-26T11:24:00.1009217Z   ]
2023-11-26T11:24:00.1009497Z }
```

![image](https://github.com/actions/first-interaction/assets/1702923/00d6e5c2-e7b8-42e1-82ce-f8e4114951b8)

I haven't changed anything about how I use `actions/first interaction` between these two runs. I'm sorry if it's spam, but it looks like a real bug to me.

**Comment by Zeal-L** [comment]
same 
![image](https://github.com/actions/first-interaction/assets/72005386/8cae0d4d-c5c6-4397-8c89-01a5d5674d0f)

**Comment by ErikBjare** [comment]
Same: https://github.com/ActivityWatch/activitywatch/actions/runs/6981763714/job/18999580470?pr=976

Not sure what the cause could be. Suddenly began failing this week.

**Comment by neilime** [comment]
Same : https://github.com/hoverkraft-tech/ci-github-common/actions/runs/6994653858/job/19028663276?pr=172

**Comment by ricardolsmendes** [comment]
Same: https://github.com/ricardolsmendes/aws-glue-ci-cd-blueprint/actions/runs/6997216504/job/19033945390?pr=34

**Comment by konsalex** [comment]
The Node version used in the Action is the same, so I guess something changed in the GitHub's Action runner?

**Comment by tuhaihe** [comment]
Same: https://github.com/cloudberrydb/cloudberrydb/actions/runs/7003830519/job/19050320906.

I tried different first-interaction versions, but it still failed.

**Comment by palisadoes** [comment]
Same here
- https://github.com/PalisadoesFoundation/talawa/pull/2168#issuecomment-1829088887

**Comment by toadfansboy** [comment]
Same here 


https://github.com/pleisto/flappy/pull/150

**Comment by palisadoes** [comment]
@pje
@cory-miller
@dakale
@damccorm

You've contributed to past releases. Please take a look

**Comment by AllanGuigou** [comment]
👋 this should be resolved with the latest release v1.3.0

**Comment by tuhaihe** [comment]
It also works for v1. Thanks @AllanGuigou!
